suitcase/json_metadata/__init__.py

```diff
@@ -67,19 +69,29 @@ def export(gen, directory, file_prefix='{uid}-', cls=event_model.NumpyEncoder,
         ones.
     **kwargs : kwargs
         kwargs to be passed to ``json.dump``.
+
     Returns
     -------
-    dest : dict
-        dict mapping the 'labels' to lists of file names
+    artifacts : dict
+        Maps 'labels' to lists of artifacts (e.g. filepaths)
+
     Examples
     --------
-    Generate files with unique-identifer names in the current directory.
+
+    Generate files with unique-identifier names in the current directory.
+
     >>> export(gen, '')
+
     Generate files with more readable metadata in the file names.
+
     >>> export(gen, '', '{plan_name}-{motors}-')
+
     Include the experiment's start time formatted as YY-MM-DD_HH-MM.
```

Comment:
Ah, we still have `YY` here. Can be fixed later, anyway.

Reply:
Thanks for checking. Just opened https://github.com/NSLS-II/suitcase-json-metadata/pull/5

Reply:
Thanks, it's a very annoying synchronization :)

Reply:
Indeed. The downside of our "many small packages" approach, but even so I still feel strongly that the benefits outweigh the costs.

Reply:
Yeah, the flexibility and individual release cycles matter.

conda-forge has the same problem, when a new feedstock repo is configured from the "current" (at some point of time) state of the staging repo, which than has to be updated if the master repo is updated.